dev-lang/luajit/luajit-2.1.1702296283.recipe

```diff
@@ -48,10 +49,12 @@ BUILD()
 INSTALL()
 {
 	make install PREFIX=$prefix INSTALL_LIB="$libDir" INSTALL_BIN="$binDir" \
-		INSTALL_INC="$includeDir" INSTALL_SHARE="$dataDir" \
-		INSTALL_MAN="$manDir/man1" INSTALL_LMOD="$dataDir/lua/5.1"
+		INSTALL_INC="$includeDir/luajit-2.1" INSTALL_SHARE="$dataDir" \
+		INSTALL_MAN="$manDir/man1" INSTALL_LMOD="$dataDir/lua/5.1" \
+		GIT_RELVER=${portVersion##*.}
```

Comment:
This fixes the location for the pkgconfig file and the lines 56 and 60 can be removed then:
```diff
-		GIT_RELVER=${portVersion##*.}
+		INSTALL_PKGCONFIG=$libDir/pkgconfig GIT_RELVER=${portVersion##*.}
```

Reply:
I am assuming you meant `$develLibDir` here?

Reply:
$developLibDir, can be used also, would obsolete fixPkgconfig (but you'd need to be sure everything is correct in the pkgconfig file then.

Reply:
I think it is correct now. `fixPkgconfig` actually breaks the file as it removes the subdirectory from the includeDirectory again (and I think we really want to install the headers into the subdirectory - this way it's also consistent with other lua packages)

Reply:
Actually, let me check again...

Reply:
Hmm... so libdir should also point to devel/lib, I guess? It doesn't for lua54 either (so that might be an issue with that package)

Reply:
This is probably where fixPkgconfig steps in, there are a few recipes around that "sed" back the subdirectory after fixPkgconfig.

Reply:
I am just fixing the libdir myself now (instead of letting fixPkgconfig fix it and then fixing the includedir breakage)